.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        device: ["xc7a35t", "xc7a100t", "xc7z010", "LIFCL-40"]
+        device: ["xc7a35t", "xc7a100t", "xc7z010", "xczu7ev", "LIFCL-40"]
```

Comment:
Unfortunately the `xczu7ev` seems to consume all memory on GH actions:

```
/home/runner/work/fpga-interchange-tests/fpga-interchange-tests/third_party/RapidWright/scripts/invoke_rapidwright.sh: line 68:  8006 Killed                  "${JAVA}" ${EXTRA_JAVA_OPTS} "$@"
```

Until we enable the custom runners I think that Kokoro will need to handle this part's testing

Reply:
Removed